test/interaction/app.js

```diff
@@ -0,0 +1,172 @@
+/* global document, window */
+
+import React, {Component} from 'react';
+import {render} from 'react-dom';
+import DeckGL, {
+  COORDINATE_SYSTEM,
+  PointCloudLayer,
+  PolygonLayer,
+  ScatterplotLayer,
+  OrbitView
+} from 'deck.gl';
+import {OrthographicView} from '@deck.gl/core';
+import {default as choropleths} from '../../examples/layer-browser/data/sf.zip.geo';
+
+let index = 0;
+window.nextTestCase = function nextTestCase() {
```

Comment:
@Pessimistress because this function is passed to puppeteer's evaluate function in node-interaction.js. It is not accessible if not attached to window.

Reply:
Okay. This should still be moved inside the App class. App should not be accessing global variables.